### PR TITLE
Extend InsertAddonRepository for Operations/Transport/Equiment Events

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: src/main/java/org/dcsa/core/events/service/OperationsEventService.java [hunk 1]
```diff
@@ -1,8 +1,9 @@
 package org.dcsa.core.events.service;
 
 import org.dcsa.core.events.model.OperationsEvent;
+import org.dcsa.core.repository.InsertAddonRepository;
 import reactor.core.publisher.Mono;
 
-public interface OperationsEventService extends EventService<OperationsEvent> {
+public interface OperationsEventService extends EventService<OperationsEvent>, InsertAddonRepository<OperationsEvent> {
```
Comment (nt-as) [inline src/main/java/org/dcsa/core/events/service/OperationsEventService.java:7]: Mismatch; `InsertAddonRepository` is not for `Service` classes.

File: src/main/java/org/dcsa/core/events/service/TransportEventService.java [hunk 2]
```diff
@@ -1,9 +1,10 @@
 package org.dcsa.core.events.service;
 
 import org.dcsa.core.events.model.TransportEvent;
+import org.dcsa.core.repository.InsertAddonRepository;
 import reactor.core.publisher.Mono;
 
-public interface TransportEventService extends EventService<TransportEvent> {
+public interface TransportEventService extends EventService<TransportEvent>, InsertAddonRepository<TransportEvent> {
```
Comment (nt-as) [inline src/main/java/org/dcsa/core/events/service/TransportEventService.java:7]: Mismatch; `InsertAddonRepository` is not for `Service` classes.

**Review by nt-as — COMMENTED** [review]
Beyond the mismatched noted, I note that `ShipementEvent` is not covered but `EquipmentEvent` is.  Lets either cover all of them or only cover the ones needed for OVS (as in, be consistent).

File: src/main/java/org/dcsa/core/events/service/TransportEventService.java (hunk 2 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/core/events/service/TransportEventService.java:7]: I've removed it, but it now fails on startup with the following error: 

`Parameter 2 of constructor in org.dcsa.core.events.service.impl.PendingEventServiceImpl required a bean of type 'org.dcsa.core.events.service.GenericEventService' that could not be found.`

Any ideas? I haven't changed anything in either class.

File: src/main/java/org/dcsa/core/events/repository/ShipmentEventRepository.java [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 import org.dcsa.core.events.model.ShipmentEvent;
 import org.dcsa.core.repository.ExtendedRepository;
+import org.dcsa.core.repository.InsertAddonRepository;
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/core/events/repository/ShipmentEventRepository.java:5]: Minor: InsertAddonRepository import is redundant.

**Review by douradofrazer — APPROVED** [review]

File: src/main/java/org/dcsa/core/events/repository/ShipmentEventRepository.java (hunk 3 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/core/events/repository/ShipmentEventRepository.java]: Thought I'd removed all of them. Removed now at least.

**Review by douradofrazer — APPROVED** [review]

**Pull request merged**